src/modules/projects/components/tables/ProjectsReferralPostingsTable.tsx

```diff
@@ -0,0 +1,78 @@
+import { Chip } from '@mui/material';
+
+import { ColumnDef } from '@/components/elements/GenericTable';
+import GenericTableWithData from '@/modules/dataFetching/components/GenericTableWithData';
+import { parseAndFormatDate } from '@/modules/hmis/hmisUtil';
+import {
+  GetProjectReferralPostingsDocument,
+  GetProjectReferralPostingsQuery,
+  GetProjectReferralPostingsQueryVariables,
+  ReferralPostingFieldsFragment,
+  ReferralPostingStatus,
+} from '@/types/gqlTypes';
+
+const StatusDisplay: React.FC<{ status: ReferralPostingStatus }> = ({
+  status,
+}) => {
+  return (
+    <Chip
+      label={status.replace(/_status$/, '').replace(/_/g, ' ')}
```

Comment:
We usually do `HmisEnums.ReferralPostingStatus[status]` so that the display value is not coupled to the enum value. I think I'd prefer to keep that pattern.